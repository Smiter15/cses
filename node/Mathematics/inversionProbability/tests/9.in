100
11 83 99 6 85 82 60 78 85 68 79 53 60 57 17 56 39 9 4 16 18 24 97 100 94 69 59 69 6 7 61 15 61 65 28 2 28 67 31 65 36 46 61 58 27 69 69 9 14 50 17 95 93 99 69 14 2 43 79 10 69 87 69 93 43 23 60 70 100 31 1 58 93 80 80 69 58 47 64 85 56 100 35 98 33 5 29 56 2 12 100 62 23 70 48 27 60 88 77 82
