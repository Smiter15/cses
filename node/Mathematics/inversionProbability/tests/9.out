2359.281318
